games/s.yaml

```diff
@@ -177,6 +207,8 @@
     license: [AGPL3]
 
 - name: Sid Meier's Pirates!
+  meta:
+    genre: [action,adventure,RTS]
```

Comment:
My bad; I guess I was thinking of the 2004 version. I haven't played the 1987 version but it looks like the land combat is real time in that one.